index.js

```diff
@@ -2,19 +2,31 @@ import { on, html, removeClass, addClass } from 'domassist';
 import Domodule from 'domodule';
 
 export default class PrettyErrors extends Domodule {
-  getRules() {
-    return null;
-  }
-
   get required() {
     return {
       options: ['event']
     };
   }
 
+  getRules() {
+    return null;
```

Comment:
Lets make this return an empty array. Here and on getErrors

Reply:
Actually better to return this.errors and initialize that value on postInit to be what we saw yesterday.